tensorflow/java/src/gen/java/org/tensorflow/processor/OperatorProcessor.java

```diff
@@ -119,46 +147,291 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
     return Collections.singleton(String.format("%s.annotation.Operator", OP_PACKAGE));
   }
 
-  private void writeApi() {
-    // Generate an empty class for now and get the build working correctly. This will be changed to
-    // generate the actual API once we've done with build-related changes.
-    // TODO:(kbsriram)
-    try (PrintWriter writer =
-        new PrintWriter(filer.createSourceFile(String.format("%s.Ops", OP_PACKAGE)).openWriter())) {
-      writer.println(String.format("package %s;", OP_PACKAGE));
-      writer.println("public class Ops{}");
+  private static final Pattern JAVADOC_TAG_PATTERN = Pattern.compile("@(?:param|return|throws|exception|see)\\s+.*");
+  private static final TypeName T_OPS = ClassName.get("org.tensorflow.op", "Ops");
+  private static final TypeName T_OPERATOR = ClassName.get("org.tensorflow.op.annotation", "Operator");
+  private static final TypeName T_SCOPE = ClassName.get("org.tensorflow.op", "Scope");
+  private static final TypeName T_GRAPH = ClassName.get("org.tensorflow", "Graph");
+  private static final TypeName T_STRING = ClassName.get(String.class);
+  private static final String OP_PACKAGE = "org.tensorflow.op";
```

Comment:
Seems like `OP_PACKAGE` is used in exactly one place, would it make sense to just inline it there instead?
So:

```java
return Collections.singleton("org.tensorflow.op.annotation.Operator")
```

instead of:

```
return Collections.singletom(String.format("%s.annotation.Operator", OP_PACKAGE));
```